### ISSUE TITLE
Mock up revisions to track workflows screen in figma

### ISSUE THREAD

**Issue by codebeaker** [FredHutch/shiny-cromwell#147]
User feedback thus far has been that the 'Track Workflows' screen should offer more info about workflows. Sonu is trying out some new versions in figma and will get feedback on them

**Comment by sckott** [comment]
Thanks @codebeaker !

**Comment by sckott** [comment]
Thanks @codebeaker 

I'm strongly in favor of the cards in a list. What's the feeling from the folks you've talked to and your feeling? 

Tables do offer some advantages but it is very awkward to put buttons/etc. into tables, which is why I went away from the tables we had in the old PROOF.

**Comment by codebeaker** [comment]
I agree with you that the lists look a lot nicer and that's where my mind went immediately too. Some of the earliest feedback was for more information density, both in terms of what info about workflows is shown and just showing more workflows at once. And some of the big things users seem to want to do on that page is sort and compare workflows against each other. So based on that I started mocking up a table version, even though we were hoping to move away from that. I figured it might be worth just getting more feedback on a revised table (that is still smaller than the current version) vs. the card list. And in the table mockup I substituted links for buttons. 
Anyway very open to more discussion on this but hope this clarifies the motivation for the table mockup. Let me know if you want to talk more about it.

**Comment by sckott** [comment]
I think it makes sense to take into account what users want for sure. That's good to know about information density. At the same time I wonder if they really want information to be more dense. I guess I'm thinking about the problem of too much information and not knowing what to do next. 

I think your option 1 above for lists (aka cards) does a great job of having more density than what they currently have

Yes, I think it's worth considering the mockups of tables at least. however, I probably wouldn't want to write any code for tables unless we decide that's the way to go since lots of time has gone into the cards.
